lib/internal/modules/esm/worker.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
```

Comment:
If this is supposed to be a script instead of a module, you can put it in `lib/internal/main/` instead.

Reply:
What's different about that location? I put it here because it's exclusively for the use of ESM (loaders)

Reply:
> What's different about that location?

https://github.com/nodejs/node/blob/3803b028dda9d7f3bf06836b5ad4522ff1ae0cb3/src/node_builtins.cc#L383-L394

> If this is supposed to be a script instead of a module, you can put it in `lib/internal/main/` instead.

I tried that, but because we are requiring it from `internal/main/worker_thread`, that doesn't work.